tools/NetHook2/README.md

```diff
@@ -1,27 +0,0 @@
-# NetHook 2
```

Comment:
The build script removed the contents in the `tools` directory. Please revert this change.